components/demo/demo-snippet.js

```diff
@@ -40,14 +43,18 @@ class DemoSnippet extends LitElement {
 			}
 			.d2l-demo-snippet-demo-wrapper.fullscreen {
 				background-color: white;
+				height: fit-content;
 				inset: 0;
-				overflow: auto;
-				position: fixed;
+				position: absolute;
 				z-index: 2;
 			}
 			.d2l-demo-snippet-demo {
 				flex: 1 1 auto;
 				position: relative;
+				translate: 0;
```

Comment:
This should prevent the demos from leaking their stacking contexts outside this container.

Reply:
Suggest adding a comment, otherwise we might forget why we added this and might be inclined to think is can be removed.
```suggestion
				translate: 0; /* create stacking context to prevent demos from leaking outside their containers */
```